packages/api/test/e2e/api.calls.e2e.ts

```diff
@@ -66,9 +66,9 @@ describe('e2e api calls', () => {
     const provider = `wss://node-6711773975684325376.jm.onfinality.io/ws?apikey=${API_KEY}`;
```

Comment:
This test needs to use a specific block hash that will always be v36

Reply:
meant to raise in the original PR where this test was added but forgot

Reply:
done